browser/static/js/main_zstack.js

```diff
@@ -1189,17 +1132,18 @@ function startCaliban(filename, settings) {
         mode.clear();
         setCanvasDimensions(payload.dimensions);
         brush.refreshView();
+        displayUndoRedo();
       }, 500, 'canvasResize');
     });
 
     window.addEventListener('keydown', (evt) => {
-      mode.handle_key(evt.key);
+      mode.handle_key(evt);
```

Comment:
I needed to make this changed for Ctrl+Z support. Whether the Ctrl key is pressed is stored in `evt.ctrlKey`, so we pass only `evt.key` to the handler, we can't tell if Ctrl was also pressed.

Reply:
Oh awesome. I hadn't realized that ctrl wasn't stored in `evt.key` although I think that is the reason why zooming uses alt instead of ctrl. I would love to make more use of the ctrl key in future work for intuitive keybinds.